src/code/z_lifemeter.c

```diff
@@ -508,7 +510,11 @@ void Health_UpdateBeatingHeart(PlayState* play) {
             interfaceCtx->beatingHeartOscillatorDirection = 0;
             if (!Player_InCsMode(play) && (play->pauseCtx.state == 0) && (play->pauseCtx.debugState == 0) &&
                 Health_IsCritical() && !Play_InCsMode(play)) {
+                #ifdef NO_LOW_HEALTH_BEEP
+                Actor_SetColorFilter(&GET_PLAYER(play)->actor, 0x4000, 255, 0, 6);
```

Comment:
I'd rather there be a separate ifdef for this new behavior of the player being red, but it may be better to not do this at all for now until we can discuss it further.

Reply:
Yes, I personally just find the beeping very distracting and annoying and wanted to replace it with something more subtle, like BOTW does for example. I understand that not everyone wants the blinking though, so I will add a new ifdef for it.